.github/workflows/conda.yml

```diff
@@ -121,10 +121,21 @@ jobs:
       - name: Install the Conda package (Linux/macOS)
         if: runner.os != 'Windows'
         run: conda install --channel conda-forge --channel ./build/conda khiops-core
-      - name: Test that the executables are installed
+      - name: Test that the executables are installed (Linux/macOS)
+        if: runner.os != 'Windows'
+        run: |
+          khiops_env --env
+          khiops -s
+          khiops -s
+      - name: Test that the executables are installed (windows)
+        if: runner.os == 'Windows'
+        shell: cmd /C call {0}
         run: |
-          MODL -v
-          MODL_Coclustering -v
+          set "KHIOPS_PROC_NUMBER=4"
```

Comment:
Why don't we need this on UNIX?

Reply:
If we set this variable, then we don't test that the new binary `_getprocnumber` works, do we?

Reply:
it's an oversight, it is to force khiops_env to set up mpi

Reply:
Ok, so this needs to be set on UNIXes as well. On the other hand, IMHO we also need to have it unset in some testing configuration, so that we can validate that the new executable is correctly used from the `khiops_env` script, in _all_ environments.